### DISCUSSION TITLE
My INPUT is a flashing LED signal on GPIO4 on an ESP32

### DISCUSSION THREAD

**Discussion by wwyyrrmm** [arendst/Tasmota#20680] [Support request]
electronic is working fine. The GPIO4 has an external pullup.
on Tasmota device, an esp32 the GPIO4 is defined as switch_n (1) with a relay (1), switchmode1 is 0
Tasmota is sending MQTT messages when the pin gets HIGH and LOW,
The problem is, that on one event the signal flashes 50 times in 8 seconds, ist is very much traffic for the mqtt, I only need ONE event, maybe to for OFF

Can anybody help me?
thx - Carsten

**Comment by sfromis** [comment]
One possibility is if you are doing something weird with rules or other automation causing actions you were not expecting. Do watch the log closely, after having activated `Weblog 3` for details.

It might also be badness in your switch or electrical setup, including long wires being an issue to be mitigated. Six times a second does not sound like mere debouncing, but if your voltages are not properly high/low (3.3V), you may get trouble, if your electronic setup is not "working fine".

**Reply by wwyyrrmm** [reply]
the electrical behavior is ok, I am getting 50 high lows in 8 seconds, this is ok, because I am getting the signal from a flashing LED.

But I do not want to send these 80 signals to my mqtt server.
